pipelines/README.md

```diff
@@ -64,15 +63,54 @@ oc create -f azureml-container-pipeline/azureml-container-pipelinerun-tensorflow
 - Apply build-secret. E.g.:
 
 ```bash
-oc apply -f <downloaddir>/rhoai-edge-build-secret.yml 
+oc apply -f <downloaddir>/rhoai-edge-build-secret.yml
 oc secret link pipeline rhoai-edge-build-pull-secret
 ```
 
 ### Deploy and run the test pipeline
 
-> **NOTE** Make sure to change the `target-imagerepo` parameter to match the name of your Quay namespace if using one of the provided `PipelineRun` files.
+> **NOTE**
+> Make sure to change the `target-imagerepo` parameter to match the name of your Quay namespace if using one of the provided `PipelineRun` files.
+
+```bash
+oc apply -k tekton/test-mlflow-image-pipeline/
+oc create -f tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow-housing.yaml
+```
+
+## Deploy the GitOps pipeline
+
+### Git Repository and Credentials
+
+- Clone/Mirror this repository on your Git server
+- Change the provided `gitops-git-user-` files to match your Git credentials
+
+### Deploy the GitOps pipeline
 
 ```bash
-oc apply -k test-mlflow-image-pipeline/
-oc create -f test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow-housing.yaml
+oc apply -k tekton/gitops-update-pipeline/
+```
+
+#### Run the GitOps pipeline
+
+The pipeline can be run through the OpenShift Pipelines web console, by specifying all of the details for your environment.
+
+Alternatively, the `tekton/gitops-update-pipeline/example-pipelineruns/` contains some examples that can be modified and used.
```

Comment:
The other pipeline directories contain the pipelinerun files directly, not under a subdirectory. Are we planning to enforce this approach across all of the pipelines?

Reply:
I'm not aware of any plans to _enforce_ anything. #62 is the closest thing I know of which plans to address how things are organised here generally. For the case of pipelinerun files, given that they're just optional examples, I think it's worth separating them from the stuff that's always expected to be provisioned.

I'm also not fully convinced that we should even include PipelineRun files in this repo. I would probably have chosen to just document (and demo) running the pipelines through the UI. I'm not sure which approach is better though.

Let me know if you'd like me to make changes to the other Pipelines to make them consistent with this, or if you want to leave it as part of #62, or if you'd like to suggest something else.

Reply:
I don't have an issue including PipelineRuns and I would say they were extremely helpful in bootstrapping the demo cluster by just changing the git server url and a few secrets.  In the long term, it would be better to standardize on a separate PipelineRuns directory that can be combined with some automation to auto-generate the environment parameter values